catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -59,6 +59,7 @@
             <argument ref="encryptionService"/>
             <property name="coordinateOrder" value="LAT_LON"/>
             <property name="metacardTypeEnhancers" ref="metacardTypeEnhancerList"/>
+            <property name="srsName" value=""/>
```

Comment:
The default in the metatype is `EPSG:4326`; any reason not to set it here as well?

Reply:
I thought about that as well. I was following the same pattern as WFS 2.0. But I can set it here if you think it's best.

Reply:
I think we should keep it consistent; and the docs are reporting the default to be `EPSG:4326`

Reply:
ok, i'll set the value